Makefile

```diff
@@ -55,6 +55,9 @@ fmt: $(GOBIN)/gofumpt
 vet:
 	go vet ./...
 
+tidy:
```

Comment:
something like `vendor` or `dependencies`?

Reply:
Thanks!
We will not use `vendor`. `go.mod` and go.sum` is enough.

Reply:
I don't mean that.

Just resolve it.